cmake/Codegen.cmake

```diff
@@ -115,10 +115,26 @@ if (INTERN_BUILD_ATEN_OPS)
     IF(MSVC)
       LIST(APPEND CPU_CAPABILITY_FLAGS "${OPT_FLAG}/arch:AVX2")
     ELSE(MSVC)
-      LIST(APPEND CPU_CAPABILITY_FLAGS "${OPT_FLAG} -mavx2 -mfma ${CPU_NO_AVX256_SPLIT_FLAGS}")
+      LIST(APPEND CPU_CAPABILITY_FLAGS "${OPT_FLAG} -mavx2 -mavx -mfma ${CPU_NO_AVX256_SPLIT_FLAGS}")
```

Comment:
Also commented on this from your avg_pool diff. Shouldn't `-mavx` already be added by the code at line 98 above?